packages/core/types-utils/tanstack/index.d.ts

```diff
@@ -1,3 +1,6 @@
 import { UseQueryOptions } from '@tanstack/react-query';
+import { AxiosResponse } from 'axios';
 
 export type UseQueryOptionsExcludedQueryKey<T, K = T> = Omit<UseQueryOptions<T, unknown, K>, 'queryKey'>;
+
+export type UseQueryOptionsWithAxios<T, K = T> = UseQueryOptionsExcludedQueryKey<AxiosResponse<T>, T>;
```

Comment:
```suggestion
export type UseQueryOptionsWithAxios<T, K = T> = UseQueryOptionsExcludedQueryKey<AxiosResponse<T>, K>;
```
이렇게 해주셔야 해요~
K가 안쓰이는중

Reply:
https://github.com/depromeet/15th-team3-FE/pull/78/commits/897cd31495c3cef0ab65f896dd709f3687eea5d3

정신머리... 감사합니다